docs/_appendix/github-intro.md

```diff
@@ -48,7 +48,7 @@ If you are a first-time GitHub user, read on for more details of the workflow.
 The test repository that you contribute to will depend on the specification
 that you are testing.
 
-**Main W3C test repository**: [github.com/w3c/web-platform-tests][main-repo]
+**Main W3C test repository**: [github.com/web-platform-tests/wpt][main-repo]
```

Comment:
Actually this isn't a problem. It'll be rendered as "Main W3C test repository: github.com/w3c/web-platform-tests" with a link.